R/mod_aboutInfo_utils.R

```diff
@@ -0,0 +1,122 @@
+
+
+
+#' Contributor Card
+#' 
+#' Generate a professional card containing role, a photo, name, site and org
+#'
+#' @param role character, role within the project, usually 'Workstream Lead',
+#'   'Core Contributor', or 'Contributor'
+#' @param pic character, file path to image, 1.3" in width
+#' @param site character, if they have a personal of GitHub site to reference
+#' @param name character, first and last name of the contributor
+#' @param org character, a organization that employes this individual, if
+#'   applicable
+#'
+#' @import shiny
+#' @importFrom bslib card card_header card_body card_image card_title
+#'
+#' @examples
+#' contrib_card(role = "Workstream Lead",
+#'              pic = 'inst/app/www/images/aaron_clark.png',
+#'              site = 'https://github.com/aclark02-arcus',
+#'              name = "Aaron Clark",
+#'              org = "Arcus Biosciences")
+#'
+#' @keywords internal
+#' @family contrib_cards
+#' 
+contrib_card <- function(role, pic, site, name, org){
+  bslib::card(
+    bslib::card_header(
+      class = "d-flex justify-content-between",
+      role
+    ),
+    bslib::card_body(
+      class = "align-items-center",
+      gap = 0,
+      fillable = TRUE,
```

Comment:
```suggestion
      fillable = FALSE,
```
This will keep the photos from getting stretched out.

Reply:
We could also probably leave this be if we standardize the size/ratio of the photos.